### DISCUSSION TITLE
about migrating from TIDB database to planetscale

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [planetscale/discussion#640] [Q&A] [answered]
Hello, I wanted to migrate from TIDB to Planetscale, The main problem that worries me is the way TIDB has stored my AUTO_INCREMENT Primary Keys they have jumps in between cuz of their distrubute architure. Like id can start like 1,2,3,4 3000, 3001, 3002, 4000,4001, 4002 so there are gaps and jumps in the Auto Increment id numbers, if I import my data into planetscale will there be any performance issue that I might see beacues of this?data is in order low to high AUTO_INCREMENT Primary Key and I didn't use any foreign key as planetscale recommends that. 
Here is a example of the ID's (Note it's in reverse sorted) AUTO_INCREMENT primary keys
![image](https://github.com/planetscale/discussion/assets/35369563/b112cae9-479f-4f21-9e73-38deb7bb83a4)

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @0xMALVEE 

No performance concerns on our end.

Gaps are not an issue. For primary keys the problem to look out for is when they aren't ordered (such as with a uuid).

Since your IDs are ordered, no performance concerns.

**Reply by 0xMALVEE** [reply]
thanks, 🥰
